koku/masu/database/report_db_accessor_base.py

```diff
@@ -111,15 +100,22 @@ def _execute_raw_sql_query(self, table, sql, bind_params=None, operation="UPDATE
         running_time = time.time() - t1
         LOG.info(log_json(msg=f"finished {operation}", row_count=row_count, table=table, running_time=running_time))
 
-    def _execute_trino_raw_sql_query(self, sql, *, sql_params=None, context=None, log_ref=None, attempts_left=0):
+    def _execute_trino_raw_sql_query(self, sql, *, sql_params=None, context=None, log_ref=None):
         """Execute a single trino query returning only the fetchall results"""
         results, _ = self._execute_trino_raw_sql_query_with_description(
-            sql, sql_params=sql_params, context=context, log_ref=log_ref, attempts_left=attempts_left
+            sql, sql_params=sql_params, context=context, log_ref=log_ref
         )
         return results
 
+    @retry(retry_on=TrinoQueryError)
```

Comment:
Is this exception type specific enough? I don't think it is.

Reply:
`TrinoQueryError` is broad and meant to catch most Exceptions from Trino. That can definintely be narrowed.